source/partials/_footer.slim

```diff
@@ -50,7 +50,7 @@ footer
     .Footer-credits
       span.Footer-handcrafted Handcrafted by
       span.Footer-subvisual SUBVISUAL
-      span.Footer-copyright © 2015 Copyright by Subvisual
+      span.Footer-copyright © 2016 Copyright by Subvisual
```

Comment:
how about making this dynamic, so that we only need to trigger a deploy next year, instead of going over this once again?